router.js

```diff
@@ -0,0 +1,85 @@
+const router = require("express").Router();
+const dbHelper = require("./data/dbHelper");
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+const env = require("dotenv");
+env.config();
+
+router.get("/", (req, res) => {
+  res.status(200).json({ message: "The router at /api is working" });
+});
+
+router.get("/users", (req, res) => {
```

Comment:
this is the endpoint that should have some protected middleware on it to check if the user is authenticated and can access these routes